src/test/java/Homework19.java

```diff
@@ -0,0 +1,37 @@
+import com.fasterxml.jackson.databind.JsonSerializable;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest{
+    @Test
+
+    public void deletePlaylist () throws InterruptedException {
+        String deletedPlayListMsg = "Deleted Playlist";
+
+        provideEmail("erika.olsen@testpro.io");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        openPlaylist();
+        clickDeletePlaylistBtn();
+        Assert.assertTrue(getDeletedPlaylistMsg().contains(deletedPlayListMsg));
+    }
+
+   public void openPlaylist() {
+       WebElement clickTestProPlaylist = driver.findElement(By.cssSelector("[href='#!/playlist/64251']"));
```

Comment:
Your defined CSS selector was unable to find the WebElement.
 `WebElement clickTestProPlaylist = driver.findElement(By.cssSelector("section[id='playlists']>ul>li:nth-child(3)"));`

Reply:
Can you explain to me how you got i:nth-child(3)"))?
I can see up to that point.
![Screenshot 2023-07-18 at 10 29 27 AM](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/137566317/708262ff-44c8-4ca3-8393-2d807f35b5f6)